common/eth2/src/lib.rs

```diff
@@ -1291,7 +1291,7 @@ impl BeaconNodeHttpClient {
         graffiti: Option<&Graffiti>,
         verify_randao: Option<bool>,
     ) -> Result<ForkVersionedResponse<BeaconBlock<T, Payload>>, Error> {
-        let mut path = self.eth_path(V2)?;
+        let mut path = self.eth_path(V1)?;
```

Comment:
Sneaky bugfix, the spec says this should be V1, our beacon HTTP server will accept any version so this may have kept us from working cross client